server/xpub-controller/helpers/files.js

```diff
@@ -127,6 +126,14 @@ class FilesHelper {
         filename,
       })
     }
+
+    const semanticExtractionEntity = new SemanticExtractionModel({
```

Comment:
I'd suggest creating factory methods for all the different field types. That way we don't expose field as a string - rather its a method like `SemanticExtractionModel.createTitleEntry(manuscriptId, title)`